packages/core/src/tracing/instrumentation/database/ioredis.js

```diff
@@ -65,10 +65,10 @@ function instrumentSendCommand(original) {
       const parentSpanSubCommands = (parentSpan.data.redis.subCommands = parentSpan.data.redis.subCommands || []);
       parentSpanSubCommands.push(command.name);
     } else if (
-      (parentSpan && constants.isExitSpan(parentSpan)) ||
+      (!skipExitTracingResult.allowRootExitSpan && parentSpan && constants.isExitSpan(parentSpan)) ||
```

Comment:
@abhilash-sivan Could you please add a comment **why**?

Reply:
Added 👍